extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -182,20 +199,70 @@ export const ManageAssetRows = ({
     return found.length > 0;
   };
 
-  const handleBlockedDomain = (assetData: AssetRowData) => {
-    dispatch(showBlockedDomainWarning());
-    setScamAssetData(assetData);
+  const checkForSuspiciousAsset = async (
+    code: string,
+    issuer: string,
+    domain: string,
+  ): Promise<NewAssetFlags> => {
+    // check revocable
+    let isRevocable = false;
+    try {
+      const resp = await server.assets().forCode(code).forIssuer(issuer).call();
+      isRevocable = resp.records[0]
+        ? resp.records[0].flags.auth_revocable
+        : false;
+    } catch (e) {
+      console.error(e);
+    }
+
+    // check if new asset
+    // TODO: stellar expert asset rating endpoint giving 403, resolve with OL
+    const isNewAsset = false;
```

Comment:
I reached out to OL, the asset `/rating` endpoint is giving me a 403 (eg https://api.stellar.expert/explorer/testnet/asset/A-GD4PLJJJK4PN7BETZLVQBXMU6JQJADKHSAELZZVFBPLNRIXRQSM433II/rating)
but only from a react app, not from the browser url so I think something with CORS going on